README.md

```diff
@@ -1,2 +1,27 @@
 # Dockit
-Dockerize it (soon™)
+Dockerize it
+
+This is still Work in Progress
+
+## How to install?
+1.) First ensure you have the latest php (>= 7.1) installed:
+```bash
+brew install php@7.2
```

Comment:
It could be a nice feature to give the user a choice on what PHP version they would like to use?

Reply:
The comment above is out of date then (>= 7.1)